sdk/translation/ci.mgmt.yml

```diff
@@ -0,0 +1,31 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
Was this copied here by mistake? I think you need to add a similar file to this [one](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/cognitivelanguage/ai-language-text/tests.yml).

Reply:
Shall I delete whole file?

Reply:
Yeah I think it is not needed. You need one similar to the one I linked to.

Reply:
It is gone.